shopelectro/context.py

```diff
@@ -0,0 +1,32 @@
+import typing
+
+from catalog.context import AbstractProductsListContext
+from catalog.models import ProductQuerySet
+from images.models import Image
+
+
+class ProductImages(AbstractProductsListContext):
+
+    @property
+    def images(self) -> typing.Dict[int, Image]:
```

Comment:
it is better off being a method and not a property, because it perform heavily computations

Reply:
@artemiy312 , we already have `tags`, `products`, `product_pages` in the same property-style.
Let's leave as is too keep it's style. And https://github.com/fidals/refarm-site/pull/194 with it's subtasks will solve this problem too